finalPrep/modZeroHero.js

```diff
@@ -2,52 +2,116 @@
 
 // Declare two variables - heroName AND specialAbility - set to strings
 
+var heroName = 'The Amazing Dad Man';
+var specialAbility = 'make egg-in-a-basket with no notice';
+
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
 
+var greeting = "Hello, I'm Dad Man!";
+var catchphrase = 'Knock that off!';
+
 // Declare two variables - power AND energy - set to integers
 
+var power = 15;
+var energy = 3;
+
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
 
+var fullPower = power * 500;
+var fullEnergy = energy + 150;
+
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
 
+var isHuman = true;
+var identityConcealed = false;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
 
+var archEnemies = ['dishes', ' bedtime', ' clear legos on the carpet'];
+var sidekicks = ['Will', 'Hazel', 'Hank'];
+
 // Print the first sidekick to your console
 
+console.log('sidekick #1', sidekicks[0]);
+
 // Print the last archEnemy to the console
 
+console.log('last archenemy', archEnemies[2]);
```

Comment:
How would you print the last `archEnemy` if you didn't know the length of the array?